content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Updating configuration via a ConfigMap
```

Comment:
(tiny nit) page titles should be in title case, according to the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/)

This feedback shouldn't block a merge.